test/unit/limited-pool.test.js

```diff
@@ -47,6 +47,18 @@ describe('LimitedPool', function() {
         });
     });
 
+    it('should trigger browser requests queue check if failed to receive browser from underlying pool', function() {
```

Comment:
"should launch next browser from queue on fail to receive browser from underlying pool"?